### DISCUSSION TITLE
如何像电脑端下边一样查看链接的网址及状态？

### DISCUSSION THREAD

**Discussion by Lanmer** [2dust/v2rayNG#3330] [General]
如何像电脑端下边一样查看链接的网址及状态？ 比如www.google.com proxy
www.amazon.cn direct 
这种

**Comment by wukan1986** [comment]
右上角菜单->Logcat

**Reply by Lanmer** [reply]
是我的设置不对吗，我的不显示具体访问的网址

**Reply by wukan1986** [reply]
好吧，我理解错了，原来你要的是域名，而不是找到不日志

**Comment by Lanmer** [comment]
![图片](https://github.com/user-attachments/assets/0db79dab-dbcc-4ff6-a6f6-01804ab81d73)
类似这种的 手机端有这个功能吗
